test/asami/multi_graph_test.cljc

```diff
@@ -167,6 +170,28 @@
     (is (empty? r8))
     (is (empty? r9))))
 
+(deftest test-txn-values
+  (let [g (-> empty-multi-graph
+              (assert-data (take 2 data) 1)
+              (assert-data (drop 2 data) 2))
+        last-stmt-id (count data)]
+    (is (= (inc last-stmt-id) (:next-stmt-id g)))
+    ;; TODO use actual queries instead once they're supported
```

Comment:
This layer doesn't understand joins, so queries aren't really the way to unit test. That would be integration tests that include multi_graph.